spec.html

```diff
@@ -1913,7 +1913,7 @@ <h1>Number::toString ( _x_ )</h1>
           </emu-note>
           <emu-note>
             <p>Implementers of ECMAScript may find useful the paper and code written by David M. Gay for binary-to-decimal conversion of floating-point numbers:</p>
-            <p>Gay, David M. Correctly Rounded Binary-Decimal and Decimal-Binary Conversions. Numerical Analysis, Manuscript 90-10. AT&amp;T Bell Laboratories (Murray Hill, New Jersey). November 30, 1990. Available as
+            <p>Gay, David M. Correctly Rounded Binary-Decimal and Decimal-Binary Conversions. Numerical Analysis, Manuscript 90-10. AT&amp;T Bell Laboratories (Murray Hill, New Jersey). 30 November 1990. Available as
```

Comment:
if we're going to normalize on a format, why not YMD?

Reply:
Because they're not fully numeric.

Reply:
and why normalize on `30 November 1990` instead of `November 30, 1990`?

Reply:
Arbitrary choice. I guess it's better because it's still in order?

This article shows many more people use big-endian/little-endian ordering than use middle-endian ordering: https://en.wikipedia.org/wiki/Date_format_by_country.

Reply:
Sure, but that ordering applies to triplicate forms like `Y/M/D` or `Y-M-D`, whereas "M D, Y" is prose - iow, the DMY/MDY ambiguity doesn't apply when the month name is non-numeric, so there doesn't seem to be a clear advantage in that regard.

To me, the "30 November 1990" form feels stilted in a sentence and like it belongs in a bibliography, where the "November 30, 1990" form feels like a better fit for a sentence.

Reply:
![image](https://user-images.githubusercontent.com/218840/94501388-f4f45780-01be-11eb-889d-4a1f644212d1.png)

Reply:
Right, but this is in prose, where none of those scenarios applies.